students/rachel_s/session04/kata14.py

```diff
@@ -24,11 +37,51 @@ def build_trigrams(words):
         if not pair in trigrams:
             trigrams[pair] = [follower]
         else:
-            trigrams[pair] += [follower]
-
+            trigrams[pair].append(follower)
```

Comment:
Minor: Lines 37-40 can be simply `trigrams.setdefault(pair, []).append(follower)`.

Or even better would be defining `trigrams = defaultdict(list)` (`from collections import defaultdict` at the top of the script), and then just say `trigrams[pair].append(follower)`.